### PR TITLE
Add support for draft prs

### PR DESCRIPTION
**Pull request by lfkwtz**

### CODE REVIEW

**Comment by lfkwtz** [comment]
chose to go with `'y'` / `'n'` since `getInput` returns a string to keep it clearly non-boolean - but open to `'true'` / `'false'` or any preference of yours

File: dist/index.js [hunk 1]
```diff
@@ -1988,6 +1988,7 @@ async function run() {
     const githubToken = core.getInput("GITHUB_TOKEN", { required: true });
     const pullRequestTitle = core.getInput("PULL_REQUEST_TITLE");
     const pullRequestBody = core.getInput("PULL_REQUEST_BODY");
+    const pullRequestIsDraft = core.getInput("PULL_REQUEST_IS_DRAFT");
```
Comment (TreTuna) [inline dist/index.js:1991]: I would rather accept a boolean for the env variable. Considering that `core.getInput` returns a string, we could convert that in the code here.

```suggestion
    const pullRequestIsDraft = core.getInput("PULL_REQUEST_IS_DRAFT").toLowerCase() === "true";
```

File: index.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ async function run() {
     const githubToken = core.getInput("GITHUB_TOKEN", { required: true });
     const pullRequestTitle = core.getInput("PULL_REQUEST_TITLE");
     const pullRequestBody = core.getInput("PULL_REQUEST_BODY");
+    const pullRequestIsDraft = core.getInput("PULL_REQUEST_IS_DRAFT");
```
Comment (TreTuna) [inline index.js:11]:
```suggestion
    const pullRequestIsDraft = core.getInput("PULL_REQUEST_IS_DRAFT").toLowerCase() === "true";
```

File: index.js [hunk 3]
```diff
@@ -37,7 +38,8 @@ async function run() {
           : `sync: ${fromBranch} to ${toBranch}`,
         body: pullRequestBody
           ? pullRequestBody
-          : `sync-branches: New code has just landed in ${fromBranch}, so let's bring ${toBranch} up to speed!`
+          : `sync-branches: New code has just landed in ${fromBranch}, so let's bring ${toBranch} up to speed!`,
+        draft: pullRequestIsDraft === 'y'
```
Comment (TreTuna) [inline index.js:42]:
```suggestion
        draft: pullRequestIsDraft
```

File: dist/index.js [hunk 4]
```diff
@@ -2017,7 +2018,8 @@ async function run() {
           : `sync: ${fromBranch} to ${toBranch}`,
         body: pullRequestBody
           ? pullRequestBody
-          : `sync-branches: New code has just landed in ${fromBranch}, so let's bring ${toBranch} up to speed!`
+          : `sync-branches: New code has just landed in ${fromBranch}, so let's bring ${toBranch} up to speed!`,
+        draft: pullRequestIsDraft === 'y'
```
Comment (TreTuna) [inline dist/index.js:2022]:
```suggestion
        draft: pullRequestIsDraft
```

File: action.yml [hunk 5]
```diff
@@ -19,6 +19,9 @@ inputs:
   PULL_REQUEST_BODY:
     description: "What you would like in the body of the pull request. Default: 'New code has just landed in {fromBranch} so let's bring {toBranch} up to speed!'"
     required: false
+  PULL_REQUEST_IS_DRAFT:
+    description: "Set to 'y' for the pull request to be opened as a draft. Default: 'n'"
```
Comment (TreTuna) [inline action.yml:23]:
```suggestion
    description: "Set to 'true' for the pull request to be opened as a draft. Default: 'false'"
```

File: README.md [hunk 6]
```diff
@@ -28,6 +28,12 @@ What you would like in the body of the pull request.
 
 Default: `sync-branches: New code has just landed in {FROM_BRANCH} so let's bring {TO_BRANCH} up to speed!`
 
+### `PULL_REQUEST_IS_DRAFT`
+
+Set to 'y' for the pull request to be opened as a draft.
```
Comment (TreTuna) [inline README.md:33]:
```suggestion
Set to `true` for the pull request to be opened as a draft.
```

File: README.md [hunk 7]
```diff
@@ -28,6 +28,12 @@ What you would like in the body of the pull request.
...
+
+Default: 'n'
```
Comment (TreTuna) [inline README.md:35]:
```suggestion
Default: `false`
```

**Pull request merged**